src/theme/separator.ts

```diff
@@ -0,0 +1,116 @@
+export default (config: { colors: string[] }) => ({
+  slots: {
+    root: 'flex items-center align-center text-center',
+    border: 'border-gray-200 dark:border-gray-800',
+    container: 'font-medium text-gray-700 dark:text-gray-200 flex',
+    icon: 'shrink-0 size-5',
+    avatar: 'shrink-0',
+    label: 'text-sm'
+  },
+  variants: {
+    color: {
+      ...Object.fromEntries(config.colors.map((color: string) => [color, { border: `border-${color}-500 dark:border-${color}-400` }])),
+      white: { border: 'border-white dark:border-gray-900' },
+      gray: { border: 'border-gray-50 dark:border-gray-800' },
+      black: { border: 'border-gray-900 dark:border-white' }
+    },
+    orientation: {
+      horizontal: {
+        root: 'w-full flex-row my-4',
```

Comment:
Radix-vue's Separator doesn't have any spacing at all, so it could feel a bit too close to other elements. Maybe `m*-4` is a bit too much, but definitely required IMO (example with them):
![image](https://github.com/benjamincanac/ui3/assets/13056429/410a0afd-3052-4f80-a2c9-870fa1d12b71)

Reply:
Discussed privately on discord:
it would be much better off letting developers decide what spacing they should use for their particular needs